File: src/utils/Level.js
function Level(xp) {
  this.xp = xp;
  this.floatLevel = Math.pow(xp, Level.K) / 3;
  this.level = Math.floor(this.floatLevel);
  this.progress = Math.floor((this.floatLevel - this.level) * 100);
}

Level.K = 1.1 / 2;

Level.fromLevel = function (level) {
  return new Level(Math.pow(3 * level, 1 / Level.K));
};

File: test/level.py
import math

p = 1.1/2

def f(xp):
	return math.pow(xp, p) / 3

def f_(level):
	return math.exp(math.log(3 * level) / p)

for i in range(1, 100):
	print("level :", i, " xp :", math.floor(f_(i)))